src/test/scala/wdl4s/SyntaxErrorSpec.scala

```diff
@@ -260,9 +260,6 @@ class SyntaxErrorSpec extends FlatSpec with Matchers {
     expectError(
       """task a {
         |  command { ./script }
-        |  output {
-        |    Int x = "bad value"
-        |  }
```

Comment:
I want to keep these WDL files to only contain the exact syntax error that they're actually testing for and nothing else.  In this case, the output section is not needed to test the `meta` section.  Worse than that though is that the output section itself contains a syntax error so it's actually masking the real syntax error of this test.  `expectError` really should probably match the error it gets back against a string so you know you got the right error.  I can file a ticket for that, because I don't want to go down too much of a rabbit hole